shared_model/interfaces/base/signable.hpp

```diff
@@ -116,10 +122,14 @@ namespace shared_model {
             .init("Signable")
             .append("created_time", std::to_string(createdTime()))
             .appendAll(signatures(),
-                       [](auto &signature) { return signature->toString(); })
+                       [](auto &signature) { return signature.toString(); })
             .finalize();
       }
 
+     protected:
+      using SignatureSetType =
+          std::unordered_set<types::SignatureType, SignableHash>;
```

Comment:
maybe we can remove PolymorphicWrapper in types::SignatureType here?

Reply:
Not that easy